blocks/identity-block/features/header-account-action/styles.scss

```diff
@@ -1,23 +1,86 @@
 .xpmedia-subs-header {
   align-items: center;
-  color: #fff;
+  color: $ui-btn-white-color;
   font-size: 0.875rem;
 
   &--button {
     align-items: center;
     background: transparent;
     border: 1px solid rgba(255, 255, 255, 0.5);
-    border-radius: 2px;
-    color: #fff;
-    font-size: 0.875rem;
+    border-radius: 4px !important;
+    color: $ui-btn-white-color;
     font-weight: bold;
     height: 2rem;
     padding: 0 0.5rem;
     text-decoration: none;
 
     &:hover {
-      color: #fff;
-      border: 1px solid #fff;
+      color: $ui-btn-white-color;
+      border: 1px solid $ui-btn-white-color;
+    }
+  }
+
+  .account-button {
+    background-color: $ui-btn-white-color;
+    transition: 500ms;
+
+    .account-text {
+      padding: 0 8px;
+    }
+
+    &:hover {
+      filter: brightness(0.85);
+      cursor: default;
+    }
+
+    @media screen and (max-width: map-get($grid-breakpoints, 'md')) {
+      right: 16px;
+      position: absolute;
+      margin-left: 16px;
+
+      .account-text {
+        display: none;
+      }
+
+      .submenu-caret {
+        display: none;
+      }
+    }
+  }
+
+  .account-menu {
+    display: inline;
+    position: absolute;
+    width: 211px;
+    height: 88px;
+    top: 50px;
+    right: 20px;
+    padding: 16px;
+    border-radius: 4px;
+    box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.25);
+    border: solid 1px #dadada;
+    font-size: 0.875rem;
+    background-color: $ui-btn-white-color;
+
+    .menu-item {
+      width: 179px;
+      margin: 0 0 8px;
+      font-size: 0.875rem;
+      line-height: 1.5;
+      text-align: left;
+
+      &-link {
+        text-decoration: none;
+
+        &:hover {
+          text-decoration: underline;
+          cursor: default;
+        }
+      }
+    }
+
+    @media screen and (max-width: map-get($grid-breakpoints, 'md')) {
+      right: 16px;
```

Comment:
same here. does this work on phones of all sizes as expected. can you use flex-based or some relative positioning here rather than hard-coding it against the right side

Reply:
this is the breakpoint we are using for the mobile version of our buttons ins some places. our designs indicate a right margin of 16px and considering the consistency of the size of the parent elements, this should be fine. the logo alignment will need looking at though.